website/docs/docs/use-dbt-semantic-layer/quickstart-semantic-layer.md

```diff
@@ -0,0 +1,308 @@
+      
+## Troubleshooting
+
+If you're encountering some issues when defining your metrics or setting up the dbt Semantic Layer, check out a list of answers to some of the questions or problems you may be experiencing.
+    
+<details>
+  <summary>How are you storing my data?</summary>
+  <div>
+    <div>dbt does not store your data, we temporarily ingest the data from the cloud data platform, pass it to connecting tool, and then drop the data.</div>
+  </div>
+</details>
+<details><summary>Is the dbt Semantic Layer open source?</summary>
+  <div>
+    <div>Some components of the dbt Semantic Layer are open source like dbt-core, the dbt_metrics package, and the BSL licensed dbt-server. The dbt Proxy Server (what is actually compiling the dbt code) and the Metadata API are not open source. <br></br>
+      
+During Public Preview, the dbt Semantic Layer is open to all dbt Cloud tiers (Developer, Team, and Enterprise). <br></br>
+      
+Team and Enterprise accounts will be able to set up the Semantic Layer and Metadata API in the integrated partner tool to import metric definition.<br></br> 
+     
+Developer accounts will be able to query the Proxy Server using SQL, but will not be able to browse pre-populated dbt metrics in external tools, which requires access to the Metadata API.</div></div>
+</details>
+<details>
+    <summary>The <code>dbt_metrics_calendar_table</code> does not exist or is not authorized?</summary>
+  <div>
+    <div>All metrics queries are dependent on either the <code>dbt_metrics_calendar_table</code> or a custom calendar set in the users <code>dbt_project.yml</code>. If you have not created this model in the database, these queries will fail and you'll most likely see the following error message:
+
+<code>Object DATABASE.SCHEMA.DBT_METRICS_DEFAULT_CALENDAR does not exist or not authorized.</code><br></br>
+
+<b>Fix</b>
+      
+<span>&#8226;</span> If developing locally, run <code>dbt run --select dbt_metrics_default_calendar</code>
+<span>&#8226;</span> If you are using this in production, make sure that you perform a full <code>dbt build</code> or <code>dbt run</code>. If you are running specific <code>selects</code> in your production job, then you will not create this required model.
+    </div>
+  </div>
+</details>
+<details>
+  <summary>Ephemeral Models - Object does not exist or is not authorized</summary>
+  <div>
+    <div>Metrics cannot be defined on <a href="/docs/building-a-dbt-project/metrics">ephemeral models</a> because we reference the underlying table in the query that generates the metric so we need the table/view to exist in the database. If your table/view does not exist in your database, you'll likely see this error message:
+
+ <code>Object 'DATABASE.SCHEMA.TESTING_EPHEMERAL does not exist or not authorized.</code><br></br>
+
+<b>Fix:</b>
+
+<span>&#8226;</span> You will need to materialize the model that the metric is built on as a table/view/incremental.</div>
```

Comment:
The `</div>` at the end of this line is also causing an issue, it is closing the list item with `</div>` instead of `</li>` thus missing a closing tag. This would be updated to:

```suggestion
- You will need to materialize the model that the metric is built on as a table/view/incremental.

</div>
```

Reply:
Note how the `</div>` is bumped down two lines